### PR TITLE
sp/gp diss - completing party editable for sbc staff

### PR DESCRIPTION
**Pull request by kialj876**
*Issue #:* /bcgov/entity#13822

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: src/views/DissolutionFirm/DissolutionFirm.vue [hunk 1]
```diff
@@ -361,6 +362,20 @@ export default class DissolutionFirm extends Mixins(DateMixin) {
...
+    console.log(currentAccount)
+    if (!currentAccount) return false
+    try {
+      console.log(JSON.parse(currentAccount).accountType)
+      return JSON.parse(currentAccount).accountType === AccountTypes.SBC_STAFF
+    } catch (error) {
+      console.log('Error parsing current account =', error)
+      return false
+    }
+  }
```
Comment (severinbeauvais) [inline src/views/DissolutionFirm/DissolutionFirm.vue:377]: As in Edit UI, this already exists in App.vue.

Just add a new getter to state-getters.ts.

File: src/store/getters/state-getters.ts [hunk 2]
```diff
@@ -100,8 +100,7 @@ export const isGovAccountUser = (state: StateIF): boolean => {
 
 /** Whether the user is SBC Staff (which is not the same as Staff). */
 export const isSbcStaff = (state: StateIF): boolean => {
-  const orgInfo = getOrgInformation(state)
-  return (orgInfo?.orgType === 'SBC_STAFF')
+  return state.stateModel.accountInformation?.accountType === AccountTypes.SBC_STAFF
```
Comment (severinbeauvais) [inline src/store/getters/state-getters.ts:103]: You might as well use `getAccountInformation` getter here instead of dereferencing the state.

Comment (severinbeauvais) [reply src/store/getters/state-getters.ts:103]: And maybe move this getter near `isPremiumAccount` in this file to keep account getters together.

Comment (kialj876) [reply src/store/getters/state-getters.ts:103]: sure

**Review by severinbeauvais — APPROVED** [review]
Ok to merge after cleanup.

**Comment by severinbeauvais** [comment]
Use getAccountInformation instead of getOrgInformation ?

**Comment by kialj876** [comment]
> Use getAccountInformation instead of getOrgInformation ?

sure. This is why shared composables are nice haha. One place with the logic for everywhere instead of all these different things that have a lot of the same data

**Pull request merged**